index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="./assets/styles/style.css" />
+    <title>Codeeq Microverse Personal Portfolio Project</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="my-logo">My Logo</h1>
+      <div class="btn-scale">
+        <img src="./assets/images/Union.svg" alt="nav icon" />
+      </div>
+    </header>
+    <main>
+      <section class="hero">
+        <h2>Hey There. I’m Sodeeq.</h2>
+        <h3>I'm a Software Developer</h3>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+
+        <article class="icons">
+          <a href="#" class="btn-scale">
+            <img src="./assets/images/Icon -GitHub.svg" alt="nav icon" />
+          </a>
+          <a href="#" class="btn-scale">
+            <img src="./assets/images/Icon - Linkedin.svg" alt="nav icon" />
+          </a>
+          <a href="#" class="btn-scale">
+            <img src="./assets/images/Icon - Angelist.svg" alt="nav icon" />
+          </a>
+          <a href="#" class="btn-scale">
+            <img src="./assets/images/Frame.svg" alt="nav icon" />
+          </a>
+          <a href="#" class="btn-scale">
+            <img src="./assets/images/Icon - Medium.svg" alt="nav icon" />
+          </a>
+        </article>
```

Comment:
- [ ] Please 🙏🏻  consider using the `ul` tag as a container and put all the social media icon links in individual `li` tags, it will make your code more professional and semantically correct. It will also help **visually-impaired** people to navigate through your website with a good *User Experience*.

- [ ] Please once you change it make sure that you must have to add styles for it in the CSS and use the flexbox there to aligned the item properly.

- [OPTIONAL] Please make sure that always close your tag with the right `tagName` otherwise as your code scale or as you will add more and more features to it, later on, it will be difficult for you to find the error just because of a *small* mistake.